learning_assistant/utils.py

```diff
@@ -22,21 +25,27 @@ def _estimated_message_tokens(message):
     return int((len(message) - message.count(' ')) / chars_per_token) + json_padding
 
 
-def get_reduced_message_list(system_list, message_list):
+def get_reduced_message_list(prompt_template, message_list):
     """
     If messages are larger than allotted token amount, return a smaller list of messages.
     """
-    total_system_tokens = sum(_estimated_message_tokens(system_message['content']) for system_message in system_list)
+    total_system_tokens = (
+        _estimated_message_tokens(prompt_template)
+        + _estimated_message_tokens('.' * 40)  # average number of characters per course name is 40
```

Comment:
Could you leave a comment about why we're estimating these and including them in the count, please?